cartridge/remote-control.lua

```diff
@@ -290,6 +297,7 @@ local function communicate(s)
 end
 
 local function rc_handle(s)
+    assert(s ~= nil, debug.traceback())
```

Comment:
Is there a new case when `s` could be `nil`?

Reply:
fixed